scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
```

Comment:
I don't think `./scripts/cibuild` is the appropriate place for running linting and tests. A more complete breakdown of what runs where might look like:
- ./scripts/setup
  - `vagrant up --provision`
- ./scripts/update
  - `npm install`
- ./scripts/test
  - lint
  - lint:ci
  - test
- ./scripts/cibuild
  - build:prod
- ./scripts/cipublish
  - `s3_website push`